package.json

```diff
@@ -75,6 +75,9 @@
     ],
     "all": true
   },
+  "peerDependencies": {
+    "semantic-release": ">=13.0.0"
```

Comment:
Imo this is of little help, because if v14 is released with a breaking change that makes it incompatible with this version (as it is the case with v3 and semantic-release v12), the version I have installed still claims to be compatible and npm doesn't warn me about upgrading. This would need to be `^13.0.0` and (if possible) updated on the next breaking change to `^13.0.0 || ^14.0.0` (or `>=13.0.0 <15.0.0`) etc.

Reply:
Yep you're right. That will require a bit of maintenance though as for each major release of the core we would have to update the `peerDependencies` of each plugins in the orga.
The pace of major releases is quite fast, but it's probably 1 every 2 months on average so it's not that big of a task.